File: Day_0_ challenge.py
import math

#Problem_1: 3 input sum finder
a, b, c = map(int, input("Enter 3 numbers: ").split())

print(a+b)
print(b+c)
print(c+a)


#Problem_2: Area calculator
r = float(input("Enter the Radius: "))
print("Area : " +  str(math.pi * r *r))


#Problem_3: Odd or Even finder
number = int(input("Enter a number: "))
print("Even" if number%2 == 0 else "Odd")

File: Day_0_challenge.java
import java.util.Scanner;
import java.lang.Math.*;

class Day_zero {
    public static void main(String[] args) {
        //Problem_1: sum finder for 3 number
        Scanner ip = new Scanner(System.in);
        System.out.println("Enter the 3 numbers: ");
        int a = ip.nextInt();
        int b = ip.nextInt();
        int c = ip.nextInt();
        System.out.println(a+b);
        System.out.println(c+b);
        System.out.println(a+c);
        
        //Problem_2: Area finder
        System.out.println("\nEnter the radius: ");
        float r = ip.nextFloat();
        System.out.println("Area : " + (Math.PI*r*r));
        
        //Problem_3: Odd or Even Finder
        System.out.println("\nEnter a number: ");
        int n = ip.nextInt();
        if(n % 2 == 0){
             System.out.println("Even");
        }
        else{
             System.out.println("Odd");
        }
    }
}
